lib/main.dart

```diff
@@ -1,12 +1,46 @@
-import 'package:farmsmart_flutter/flavors/flavor.dart';
-import 'package:farmsmart_flutter/main_common.dart';
-import 'package:farmsmart_flutter/data/firebase_const.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_localizations/flutter_localizations.dart';
 
-void main() async{
+import 'app_coordinator.dart';
+import 'farmsmart_localizations.dart';
 
-  // Write to shared preference
-  AppSettings.get().environment = (FirestoreEnvironment.DEVELOPMENT);
+class _Constants {
+  static final String fontFamily = 'GT-America-Standard';
 
-  bootstrap();
+  static final backgroundColor = Color(0xFFFFFFFF);
+  static final primaryGrey = Color(0xFF757575);
+
+  static final List<Locale> supportedLocales = [
+    const Locale('en'),
```

Comment:
could you talk to @isaacmartinezimv about hooking it up to the switch in the landing page...